fleetshard/config/config.go

```
@@ -51,6 +50,9 @@ func GetConfig() (*Config, error) {
 	if c.AuthType == "" {
 		configErrors.AddError(errors.New("AUTH_TYPE unset in the environment"))
 	}
+	if c.ManagedDBEnabled && c.AWSRoleARN == "" {
```

You could include `MANAGED_DB_SECURITY_GROUP` in this check too. It's better to fail early here, then fail in the AWS API call.

I'm undecided about `MANAGED_DB_SUBNET_GROUP`. The AWS client doesn't complain if this one's missing, if there are subnets in the VPC that it can use. But maybe we should enforce it ourselves.

---

Done.